db/fixtures/miq_product_features.yml

```diff
@@ -5129,6 +5137,10 @@
         :description: Suspend VM
         :feature_type: control
         :identifier: vm_suspend
+      - :name: Pause
+        :description: Pause VM
+        :feature_type: control
+        :identifier: vm_pause
```

Comment:
@abellotti changes look good, i don't think these affect UI at all. 
Currently UI does not have Pause VM button with button id "vm_pause" and there is no individual identifier defined in product features to be able to view list of Policies/Policy Profiles under control explorer. Currently if a user has access to "policy" feature identifier they are able to see list and summary of Policies, feature being checked for list/summary of Policy profiles is "policy_profile"